cmd/brew-services.rb

```diff
@@ -121,20 +125,39 @@ def homebrew!
       extend(FileUtils)
     end
 
-    # Access current service
-    def service; @service ||= Service.new(Formula.factory(@formula)) if @formula end
+    # Access target services
+    def target_services
+      @target_services ||= begin
+        if @formula == '--all'
```

Comment:
Was thinking it might be nice to split `@formula == '--all'` logic everywhere into a method like `def all_services?` and also perhaps make use of `ARGV.include?` and/or `ARGV.delete` instead.